src/UI/Art/SpriteArtist/SpriteInfoView.java

```diff
@@ -0,0 +1,142 @@
+import UI.COStateInfo;
+import UI.InfoController;
+import UI.MapView;
+
+/**
+ * InfoView paints the CO's full portrait and a box with info in it.
+ * The contents of the info are defined by inputs; current game status or static CO info are both used
+ */
+public class SpriteInfoView extends MapView // Extend MapView for getDrawableMap(). We don't actually draw it, but we need fog info.
+{
+  private InfoController myControl;
+  
+  /** Width of the visible space in pixels. */
+  private int viewWidth;
+  /** Height of the visible space in pixels. */
+  private int viewHeight;
+
+  public SpriteInfoView(InfoController control)
+  {
+    Dimension dims = Driver.getInstance().gameGraphics.getScreenDimensions();
+    viewWidth = dims.width;
+    viewHeight = dims.height;
+    myControl = control;
+  }
+
+  @Override
+  public Dimension getPreferredDimensions()
+  {
+    return new Dimension(viewWidth, viewHeight);
+  }
+
+  @Override
+  public void setPreferredDimensions(int width, int height)
+  {
+    viewWidth = width;
+    viewHeight = height;
+    // Let SpriteOptions know we are changing things.
+    SpriteOptions.setScreenDimensions(viewWidth, viewHeight);
+  }
+
+  @Override
+  public void render(Graphics g)
+  {
+    DiagonalBlindsBG.draw(g);
+
+    // Get the CO info menu.
+    int drawScale = SpriteOptions.getDrawScale();
+    
+    int paneOuterBuffer = 4*drawScale; // sets both outer border and frame border size
+    int paneHSize = (int) (viewWidth*0.7) - paneOuterBuffer*4; // width of the BG color area
+    int paneVSize = (int) (viewHeight   ) - paneOuterBuffer*4; // height ""
+
+    // Get the current menu selections.
+    Commander thisCO = myControl.getSelectedCO();
+    InfoPage page = myControl.getSelectedPage();
+
+    // Draw the commander art.
+    BufferedImage COPic = SpriteLibrary.getCommanderSprites(myControl.getSelectedCOInfo().name).body;
+    // justify bottom/right
+    g.drawImage(COPic, viewWidth - COPic.getWidth()*drawScale, viewHeight - COPic.getHeight()*drawScale,
+        COPic.getWidth()*drawScale, COPic.getHeight()*drawScale, null);
+
+    if (null != thisCO)
+      CommanderOverlayArtist.drawCommanderOverlay(g, thisCO, false);
+    
+    // add the actual info
+    g.setColor(SpriteUIUtils.MENUFRAMECOLOR); // outer buffer
+    g.fillRect(  paneOuterBuffer,   paneOuterBuffer, paneHSize + 2*paneOuterBuffer, paneVSize + 2*paneOuterBuffer);
+    g.setColor(SpriteUIUtils.MENUBGCOLOR);    // inside of the pane
+    g.fillRect(2*paneOuterBuffer, 2*paneOuterBuffer, paneHSize                    , paneVSize                    );
+    
+    // TODO: consider drawing all pages as one big image, so the user can scroll smoothly through it regardless of screen size
+    int drawingWidth = paneHSize - paneOuterBuffer;
+    switch (page.pageType)
+    {
+      case CO_HEADERS:
+        int overlayHeight = 42*drawScale;
+        int heightOffset = 0;
+        for (Commander CO : myControl.getGame().commanders)
```

Comment:
This page doesn't seem to differ based on which Commander is in focus. The info on this page is also mostly covered on the next page (with the exception of the CO panel, which is already in the top right).

Reply:
It's *all* of the CO headers at once, for a summary of the overall gamestate (and so you can see your opponents' meters).

Reply:
Sure. It just feels weird to cycle the CO in the background and have nothing else change.

I won't make you change it, but I might want to change it one day.